src/modules/Modules.tsx

```diff
@@ -6,14 +6,18 @@ import PatternsProvider from 'core/patterns';
 import TechnologiesProvider from 'core/technologies';
 
 import ModulesRouter from './ModulesRouter';
+import { useLocation } from 'react-router-dom';
```

Comment:
Invalid import order. useLocation is from 3th party deps so should be put above.

Reply:
import React from 'react';
import { useLocation } from 'react-router-dom';

import { AlertsManager } from 'core/api';
import AuthProvider from 'core/auth';
import PatternsProvider from 'core/patterns';
import TechnologiesProvider from 'core/technologies';

import { shouldLoadPatterns, shouldLoadTechnologies } from './shouldLoadDataOnInit';
import ModulesRouter from './ModulesRouter';

Czy takie coś jest ok?